sound/soc/sof/ipc4-pcm.c

```diff
@@ -835,5 +835,6 @@ const struct sof_ipc_pcm_ops ipc4_pcm_ops = {
 	.dai_link_fixup = sof_ipc4_pcm_dai_link_fixup,
 	.pcm_setup = sof_ipc4_pcm_setup,
 	.pcm_free = sof_ipc4_pcm_free,
-	.delay = sof_ipc4_pcm_delay
+	.delay = sof_ipc4_pcm_delay,
+	.trigger_ipc_first = true
```

Comment:
nit-pick: if this is limited to playback, we should have a name that reflects that. Or we change it from boolean to mask where BIT(SNDRV_PCM_STREAM_PLAYBACK) is set so that we can extend later if this was ever needed

Reply:
@plbossart I have removed the restriction for playback only and made it applicable for both playback and capture now.